pkg/target/target_integration_test.go

```diff
@@ -131,7 +131,7 @@ func TestConstraintEnforcement(t *testing.T) {
 		allowed    bool
 	}{
 		{
-			name:       "match everything",
+			name:       "match deny all",
```

Comment:
Changed since there is another "match everything" below that matches namespace, kind, and label/ns selectors